docs/elements/examples/shadcn-ui.mdx

```diff
@@ -506,3 +530,47 @@ export default function SignInPage() {
 }
 ```
 </CodeBlockDemo>
+
+
+## OTP Input
+
```

Comment:
```suggestion

The following example demonstrates how to make a one-time password (OTP) input with Clerk Elements.
```
Added this sentence because it wasn't immediately obvious what this example was